src/content/advanced/coredns/index.md

```diff
@@ -19,39 +20,18 @@ owner:
 
 Your Giant Swarm installation comes with a default configuration for the [CoreDNS addon](https://github.com/coredns/coredns)
 
-You can override these defaults in a ConfigMap named `coredns-user-values`.
+You can override these defaults in a ConfigMap named `coredns-user-values` in the management cluster.
 
 ## Where is the user values ConfigMap
 
-Given the cluster you are trying to configure has id: `123ab`
-
-**Workload cluster release v9.0.1 and greater:**
-
-If your cluster is on workload cluster release v9.0.1 or newer then you will find the `coredns-user-values` ConfigMap on the Control Plane in the `123ab` namespace:
+Given the cluster you are trying to configure has id: `123ab` then you will find the `coredns-user-values` ConfigMap in the management cluster in the `123ab` namespace:
```

Comment:
The migration from chartconfig to app CRs is complete. So we can simplify the docs.